CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+4.6.1
+---
+
+* Add Django 2.0 support
+
+ - Add on_delete=models.CASCADE in models & migrations
+ - Add Django 2.0 to tests matrixs
```

Comment:
Not a big deal at all, but English is weird, so the plural of 'matrix' is 'matrices', btw.

Reply:
@coddingtonbear , fixed.